src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -868,9 +898,13 @@ def _unpack_custom_build(self):
         os.makedirs(directory)
       gcs_path = f'/{custom_builds_bucket}/{self.custom_binary_key}'
       storage.copy_file_from(gcs_path, build_local_archive)
-    elif not blobs.read_blob_to_disk(self.custom_binary_key,
+    else:
+      gcs_path = blobs.get_gcs_path(self.custom_binary_key)
+      if not blobs.read_blob_to_disk(self.custom_binary_key,
                                      build_local_archive):
-      return False
+        return False
+
+    self._emit_build_age_metric(gcs_path)
```

Comment:
If I understand correctly, this will be emitted for all tasks.
Tasks like progression and regression that try many versions, starting with the latest to check if the crash occurs or not. I don't think this approach would work there. We rather need to just verify that what we consider the latest version on such job/task is not too old, not every version tried.

As for analyze_task, it was pointed out that the uploader can specify the version they want to test on, in that case, we should not emit a metric.